packages/rocketchat-lib/lib/configLogger.js

```diff
@@ -0,0 +1,19 @@
+/* globals LoggerManager */
+RocketChat.settings.get('Log_Package', function(key, value) {
+	return (typeof LoggerManager !== 'undefined') && LoggerManager !== null && (LoggerManager.showPackage = value);
```

Comment:
actually, if the variable `LoggerManager` have never been declared, It will throw an exception
```javascript
LoggerManager && LoggerManager.showPackage = value; // Uncaught ReferenceError: LoggerManager is not defined
```

```javascript
typeof LoggerManager != 'undefined' && LoggerManager.showPackage = value; // nothing happens
```